resources/migrations/000_migrations.yaml

```diff
@@ -11628,6 +11628,102 @@ databaseChangeLog:
                     deferrable: false
                     initiallyDeferred: false
 
+  - changeSet:
```

Comment:
1. We have a rule that forces there is only one change per changeSet, so you'll need to split these into multiple changeSet [src](https://github.com/metabase/metabase/blob/d051288963a7658d4f9351777d5f150c127ed598/bin/lint-migrations-file/src/change_set/strict.clj#L18)
2. Also, make sure you claim the id in #migrations channel to avoid conflict

Reply:
Thanks, I didn't know about that channel.
I've claimed 002 through 010 and split this into separate changesets.